contributing/commenting_fields_enums.adoc

```diff
@@ -0,0 +1,54 @@
+= Commenting on fields and enums
+
+The reference documentation of OSI is created using _Doxygen_.
+The content of the reference documentation is generated from comments in the `.proto` files.
+Follow these steps when creating comments on fields and enums to keep the documentation style consistent.
+
+**Prerequisites**
+
+* You have created a field or enum.
+
+**Steps**
+
+. Add a description to the field or enum followed by an empty comment line.
+. Add the unit of the field using the `\unit` keyword followed by an empty comment line.
+. Optionally, add literary references to the comments.
+. Optionally, add rules encapsulated between `\rules` and `endrules`.
+
+**Example**
+
+----
+// \brief The conditions of the environment.
+//
+// \image html EnvironmentalConditions.svg
+//
+// Definition of light, weather conditions and other environmental conditions.
+//
+// \note These conditions apply locally around the host vehicle.
+//
+message EnvironmentalConditions
+{
+    // Atmospheric pressure in Pascal at z = 0.0 m in world frame (about 101325 Pa) [1, 2].
+    //
+    // Unit: Pa
```

Comment:
Is this correct? Above a keyword _\unit_ is used.

Reply:
I searched through the proto files. The example appears to be correct. Corrected the above instruction.